Source/Controllers/MainViewControllers/TableContent/SPTableContent.m

```diff
@@ -527,7 +527,13 @@ - (void)setTableDetails:(NSDictionary *)tableDetails {
 
 		// Set up the column
 		theCol = [[NSTableColumn alloc] initWithIdentifier:[columnDefinition objectForKey:@"datacolumnindex"]];
-		[[theCol headerCell] setAttributedStringValue:[columnDefinition tableContentColumnHeaderAttributedString]];
+
+        if ([prefs boolForKey:SPDisplayTableViewColumnTypes]) {
+            [[theCol headerCell] setAttributedStringValue:[columnDefinition tableContentColumnHeaderAttributedString]];
+        } else {
+            [[theCol headerCell] setStringValue:[columnDefinition objectForKey:@"name"]];
+        }
```

Comment:
There probably also needs to be a key value observer set up to reload the table content when the option is changed

Reply:
> > <img alt="image" width="812" src="https://user-images.githubusercontent.com/11057501/204401696-c7cbbde7-919d-4f4a-9161-cde2bf18414e.png">
> 
> Looks like the checkbox is overlapping the line? That means the constraint on the bottom of Save Query History Individually needs to be removed and a new constraint added to the bottom of your new checkbox.
> 
> Basically the auto layout constraints need to be adjusted

@Jason-Morcos  I'm struggling to get the auto-layout to work so I think I'll need a hand with this. I have a feeling it's simple when you know how, but unfortuantely I've got no experience with Xcode or auto-layout/interface builder.

Reply:
> There probably also needs to be a key value observer set up to reload the table content when the option is changed

@Jason-Morcos I've tried adding an observer in my latest commit but it doesn't appear to be working :(. Can you see what's wrong with it? Is it possible that `[tableContentView reloadData];` only reloads the actual data of the table and not the headers?